source/usage-examples/deleteMany.txt

```diff
@@ -0,0 +1,86 @@
+.. _csharp-delete-many:
+
+=====================
+Delete Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can delete more than one document using the ``DeleteMany()`` method on
+a collection object.
+
+Example
+-------
+
+These examples use the following ``Restaurant`` class as a model:
+
+.. literalinclude:: ../includes/code-examples/DeleteMany.cs
+   :start-after: start-model
+   :end-before: end-model
+   :language: csharp
+   :copyable:
+   :dedent:
+
+The following code deletes all documents in the ``restaurants`` collection that have a 
+``name`` field which includes the word "Green".
```

Comment:
trying to make this more precise. lmk if you can make it even better

```suggestion
The following code deletes all documents in the ``restaurants`` collection whose``name`` field has a value that includes the word "Green".
```

Reply:
I shortened it a bit, but I still think the meaning is very clear. The phrase "field value" is also used elsewhere in our documentation.